### PR TITLE
fixed the build, fixed services failing to start on dev:live

### PR DESCRIPTION
**Pull request by FrozenKiwi**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,6 +20,9 @@
 **/*secret*
 **/.firebase/hosting.*.cache
 
+# ignore the IPFS instance we spin up in dev:live
```
Comment (MarieTheCoin) [inline .gitignore:23]: Comments!!! 👌

File: package.json [hunk 2]
```diff
@@ -123,7 +125,7 @@
     "chalk": "^2.4.2",
     "concurrently": "^5.1.0",
     "cpx": "^1.5.0",
-    "cross-env": "5.2.0",
+    "cross-env": "^7.0.3",
```
Comment (MarieTheCoin) [inline package.json:128]: Big version change

**Review by MarieTheCoin — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (FrozenKiwi) [reply package.json:128]: yeah, I'm actually not sure what happened here.  Some kind of merge conflict meant it got uninstalled, so I just re-installed at latest.

It's job is pretty simple: just setting env vars on startup, so... *shrugs*.

**Comment by FrozenKiwi** [comment]
Ignoring snyk - I've tried rebuilding multiple times and it has not removed that error.

**Pull request merged**